mapie/utils.py

```diff
@@ -398,9 +406,70 @@ def check_nan_in_aposteriori_prediction(X: ArrayLike) -> None:
     WARNING: at least one point of training set belongs to every resamplings.
     Increase the number of resamplings
     """
-    if np.any(np.all(np.isnan(X), axis=1), axis=0):
+    if len(X.shape) == 1:
+        if np.any(np.isnan(X)):
+            warnings.warn(
+                "WARNING: At least one point of training set "
+                + "belongs to every resamplings.\n"
+                "Increase the number of resamplings"
+            )
+    elif np.any(np.all(np.isnan(X), axis=1), axis=0):
         warnings.warn(
             "WARNING: at least one point of training set "
             + "belongs to every resamplings.\n"
             "Increase the number of resamplings"
         )
+
+
+def check_lower_upper_bounds(
+    y_preds: NDArray,
+    y_pred_low: ArrayLike,
+    y_pred_up: ArrayLike
+) -> None:
+    """
+    Perform a check on the final results to check if the
+    lower bound values or upper bound values provide a disagreement
+    with the idea that upper bounds should always be strickly greater
+    than lower bound values (but also compared to predictions at
+    quantile: 0.5)
```

Comment:
Could be this check used in `MapieRegressor` as well ? I think there is a unit test dedicated to this check but it could be worth adding this check in `MapieRegressor`. If so, one should modify it to make it more generic.